.evergreen.yml

```diff
@@ -230,63 +245,45 @@ functions:
-      remote_file: spruce/${task_id}/storybook/
-      bucket: mciuploads
-      content_type: text/html
-      permissions: public-read
-      display_name: storybook-
-  
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-css:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.css"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: text/css
-      permissions: public-read
-      display_name: storybook_assets_css_
-
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-js:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.js"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: application/javascript
-      permissions: public-read
-      display_name: storybook_assets_js_
-
-  # Ideally we can get rid of this command upon the completion of EVG-17334.
-  attach-storybook-assets-map:
-    command: s3.put
-    type: system
-    params:
-      aws_key: ${aws_key}
-      aws_secret: ${aws_secret}
-      local_files_include_filter:
-        ["spruce/storybook-static/assets/*.js.map"]
-      remote_file: spruce/${task_id}/storybook/assets/
-      bucket: mciuploads
-      content_type: application/json
-      permissions: public-read
-      display_name: storybook_assets_map_
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["storybook-static/*.html"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: text/html
+        permissions: public-read
+        display_name: storybook-
+        preserve_path: true
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["spruce/storybook-static/**/*.js", "spruce/storybook-static/**/*.mjs"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: application/javascript
+        permissions: public-read
+        display_name: storybook_assets_js_
+        preserve_path: true
+    - command: s3.put
+      type: system
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_files_include_filter:
+          ["spruce/storybook-static/**/*.js.map"]
+        remote_file: spruce/${task_id}/storybook/
+        bucket: mciuploads
+        content_type: application/json
+        permissions: public-read
+        display_name: storybook_assets_map_
+        preserve_path: true
```

Comment:
Great job using `preserve_path` flag we can probably remove this condition in the storybook config.
https://github.com/evergreen-ci/spruce/blob/5e564b68cbda0ea490c52d15425d702b50b74fc0/.storybook/main.js#L47-L49

Reply:
Interestingly removing this breaks the Storybook iframe's file paths!